README_PYPI.md

```
@@ -0,0 +1,179 @@
+<!-- AUTO-GENERATED FILE - DO NOT EDIT BY HAND.
+  PyPI long description derived from README.md by
+  scripts/generate_pypi_readme.py (make pypi-readme): every picture
+  element collapses to its light-theme img fallback and animated
+  clips to their poster stills, because PyPI's readme renderer
+  strips the picture and source tags. Edit README.md instead. -->
+
+<a href="https://jmrplens.github.io/phonometry/"><img src="https://raw.githubusercontent.com/jmrplens/phonometry/main/.github/brand/banner.webp" alt="phonometry, acoustic measurement toolkit for Python" width="100%"></a>
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)